### PR TITLE
VDYP-176 VRIStart data parsing

### PR DESCRIPTION
**Pull request by smithkm**

### CODE REVIEW

**Comment by smithkm** [comment]
#33 included a change to the formatter but did not apply the changes.  This PR did apply the formatter leading to lots of unrelated formatting changes, mostly comment text reflowing to use the larger width. I've made PR #35 that just applies those changes.  Merging that first should make this PR much more manageable for review.

File: vdyp-vri/src/test/java/ca/bc/gov/nrs/vdyp/vri/VriStart.java [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+		return vdypPolygon;
+	}
+
+	@Override
+	public VdypApplicationIdentifier getId() {
+		return VdypApplicationIdentifier.VRI_START;
+	}
+
+	@Override
+	protected BaseControlParser getControlFileParser() {
```
Comment (mjunkin) [inline vdyp-vri/src/test/java/ca/bc/gov/nrs/vdyp/vri/VriStart.java:231]: Is it intended that this returns null, or have you just not gotten to writing an implementation?

**Review by mjunkin — CHANGES_REQUESTED** [review]

File: vdyp-fip/src/main/java/ca/bc/gov/nrs/vdyp/fip/model/FipLayer.java [hunk 2]
```diff
@@ -7,41 +7,29 @@
...
 import ca.bc.gov.nrs.vdyp.model.LayerType;
+import ca.bc.gov.nrs.vdyp.model.SingleSiteLayer;
 
-public class FipLayer extends BaseVdypLayer<FipSpecies> {
+public class FipLayer extends SingleSiteLayer<FipSpecies, FipSite> {
 
 	private float crownClosure; // FIPL_1/CC_L1 or FIP:_V/CC_V1
-	private String siteSpecies; // FIPL_1A/SITESP64_L1 or FIPL_VA/SITESP64_L1
+
+	// TODO move to FipSite
```
[the remark this answers is not in the document]
Comment (smithkm) [reply vdyp-fip/src/main/java/ca/bc/gov/nrs/vdyp/fip/model/FipLayer.java:16]: Something I moved.

File: vdyp-common/src/main/java/ca/bc/gov/nrs/vdyp/application/VdypStartApplication.java [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+	protected abstract BaseControlParser getControlFileParser();
+
+	void closeVriWriter() throws IOException {
+		if (vriWriter != null) {
+			vriWriter.close();
+			vriWriter = null;
+		}
+	}
+
+	public void setControlMap(Map<String, Object> controlMap) {
```
[the remark this answers is not in the document]
Comment (smithkm) [reply vdyp-common/src/main/java/ca/bc/gov/nrs/vdyp/application/VdypStartApplication.java:141]: That was visible for some unit tests.  I've made it package visible and created a utility method to allow tests in other packages to manipulate it.

**Review by mjunkin — APPROVED** [review]

**Pull request merged**